### ISSUE TITLE
newline-after-import ignoring angular annotations.

### ISSUE THREAD

**Issue by eestein** [benmosher/eslint-plugin-import#1784]
With the following rule:

`"import/newline-after-import": "error"`

If I write:

```typescript
import { DemoModel } from '../app/models/demo.model';
import { HomeComponent } from './views/home/home.component';
@Component({
    selector: 'app-root',
    templateUrl: './app.component.html',
    styleUrls: ['./app.component.scss']
})
```

No error is throw. But if I write:

```typescript
import { DemoModel } from '../app/models/demo.model';
import { HomeComponent } from './views/home/home.component';
const x = 2;
```

it works as expected.
The way I see it, it should not care what the next line is (if not import, of course).

Here are the images:

![Screen Shot 2020-05-29 at 11 50 42](https://user-images.githubusercontent.com/2049255/83273295-be6fe700-a1a2-11ea-900d-40dbbcb02cc2.jpg)
![Screen Shot 2020-05-29 at 11 50 26](https://user-images.githubusercontent.com/2049255/83273299-c039aa80-a1a2-11ea-9a92-82ac362d286f.jpg)

**Comment by ljharb** [comment]
What is an "anchor annotation"? That looks like decorators syntax, a stage 2 proposal that's on its 4th syntax/semantics iteration.

Are you saying you expect a newline above the decorator, below the imports, and it's not forcing one? If so, that is indeed a bug.

**Comment by eestein** [comment]
> Are you saying you expect a newline above the decorator, below the imports, and it's not forcing one? If so, that is indeed a bug.

Exactly.

**Comment by OlegBrony** [comment]
also, it ignores all decorators.

**Comment by nbitouze** [comment]
I can confirm that bug makes `newline-after-imports` pretty much useless in an Angular project, considering most of the typescript files in an angular project start with imports followed directly by the declaration of a decorated class.

**Comment by notaphplover** [comment]
Any updates about this?
